infra/helper.py

```diff
@@ -124,6 +132,18 @@ def parse_args(parser, args=None):
   return parser.parse_args(args)
 
 
+def _add_external_project_args(parser):
+  parser.add_argument('--build-integration-path',
+                      help=('Path to the build integration for non-OSS-Fuzz '
+                            'projects.'),
+                      default=None)
+
+  parser.add_argument('--project-src-path',
```

Comment:
Instead of requiring both `project_name` and this, we can just support passing a directory as the project name. Can add a "--external" boolean option to distinguish this case.

Reply:
I did a major refactoring of helper.py to support this. PTAL.